src/components/EpisodesNav/EpisodesNav.jsx

```diff
@@ -1,23 +1,61 @@
 import React, { useContext } from "react";
 import { 
 	data,
+	ACTIONS,
 	EpisodesContext as Episodes
 } from "COMPONENTS/Episodes/";
-import { RENDER } from "./";
+import { s, RENDER } from "./";
 
 export default function EpisodesNav(){
 	
+	// CONTEXT
+	// -------------------------------
+	const {
+		state: {
+			activeIndex,
+			episodeCount
+		},
+		dispatch
+	} = useContext(Episodes);
+
+
+	// RENDER
+	// -------------------------------
+	const isFirstEpisodeActive = activeIndex === 0;
+	const isLastEpisodeActive  = activeIndex === episodeCount - 1;
+
 	const pagination = data.map(RENDER.pagination);
```

Comment:
The render should be the absolute last thing before the `return` - everything else is basically getting variables and values ready for this section, so there shouldn't be anything under it.